File: app/src/main/java/com/example/inktouch/InkTouchApp.java
package com.example.inktouch;

import android.content.Context;
import androidx.multidex.MultiDex;
import androidx.multidex.MultiDexApplication;

public class InkTouchApp extends MultiDexApplication {
    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        MultiDex.install(this);
    }
}

File: app/src/main/java/com/example/inktouch/InkTouchApp.kt
package com.example.inktouch

import android.app.Application
import androidx.appcompat.app.AppCompatDelegate

class InkTouchApp : Application() {
    override fun onCreate() {
        super.onCreate()
        // Enable vector drawable support for older Android versions
        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)
    }
}
